lib/plugins/aws/package/compile/events/eventBridge/index.test.js

```diff
@@ -750,6 +750,119 @@ describe('AwsCompileEventBridgeEvents', () => {
       );
     });
 
+    it('should create the necessary resources when using the default event bus arn', () => {
+      awsCompileEventBridgeEvents.serverless.service.functions = {
+        first: {
+          name: 'first',
+          events: [
+            {
+              eventBridge: {
+                eventBus: 'arn:aws:events:us-east-1:12345:event-bus/default',
+                schedule: 'rate(10 minutes)',
+                pattern: {
+                  'source': ['aws.cloudformation'],
+                  'detail-type': ['AWS API Call via CloudTrail'],
+                  'detail': {
+                    eventSource: ['cloudformation.amazonaws.com'],
+                  },
+                },
+              },
+            },
+          ],
+        },
+      };
+
+      return expect(awsCompileEventBridgeEvents.compileEventBridgeEvents()).to.be.fulfilled.then(
```

Comment:
All new unit tests should be written with `runServerless`. Check: https://github.com/serverless/serverless/tree/master/tests#unit-tests, you may use extend feature to extend basic `function` fixture and tests against that.

e.g. it's how tests were configured for latest ALB addition: https://github.com/serverless/serverless/blob/04d463f4e438d02fef958fce34677a06645fee82/lib/plugins/aws/package/compile/events/alb/lib/healthCheck.test.js

Reply:
I think I got it now, though I didn't have time to convert all the existing test cases to the runServerless method. I'll try to get around to it by tomorrow. Should I make a new pull request for that?

Reply:
Hmm, seems like the test fails on windows. There is no error message or anything to indicate what could have gone wrong :frowning_face: 

I don't have a windows machine to check this

Reply:
> Hmm, seems like the test fails on windows

It's very weird error. I've run your branch on my Windows instance and tests have run successfully. To be honest I have not clear idea what happens there (especially that there's no error log, for unknown reason).

However I've patched one issue which prevented an async function to throw a sync error, can you merge with master and and push changes again?